packages/components/src/docs/Migration/ui-slider.mdx

```diff
@@ -0,0 +1,153 @@
+# Migration Guide: From `ui-slider` to `sd-carousel`
+
+The new `sd-carousel` is designed to replace the `ui-slider`. Instead of mainly providing content via attributes, the `sd-carousel` component uses slots to allow for more flexibility and customization.
+
+## 💾 Slots
+
+### ✨ New Slots
+
+#### [default]
```

Comment:
istn't this the same as in the ui-slider? => would not be a new slot...

Reply:
Ah, you're right. Removed.